prod.yaml.jinja

```
@@ -88,19 +88,24 @@ services:
       - .docker/db-creation.env
     restart: unless-stopped
     {%- if postgres_exposed %}
+    {%- if traefik_version == 3 %}
```

AFAICS, this should work with traefik 2 too: https://doc.traefik.io/traefik/v2.11/routing/routers/#rule_1

```suggestion
    {%- if traefik_version > 1 %}
```

---

The problem with this is that Traefik 2 cannot differentiate between multiple databases by domain. You would need to create an entrypoint and open a port for each database within a server. I have tested this quite a few times.

So, we could say that it does not work in the most optimal way. That’s why I think it should currently be configured only for Traefik 3, and later we could add support for Traefik 2 and specify its limitations.

I’m currently working on the test.

---

Whoa, that's weird because it claims to support also the HostSNI router. But well, my experience is based on reading the docs; if yours is based on experimentation, then I'll trust you! Thanks for sharing.